assets/dev/scss/frontend/conditionals/with-breakpoints/lightbox.scss

```diff
@@ -1,5 +1,5 @@
-@import "../../helpers/variables";
-@import "../breakpoints/breakpoints";
```

Comment:
@mykytamurzin Can you please apply this to the following file as well?

`assets/dev/scss/frontend/conditionals/with-breakpoints/apple-webkit.scss`